cmd/scollector/main.go

```diff
@@ -245,17 +245,27 @@ func main() {
 		}
 		collect.MaxQueueLen = conf.MaxQueueLen
 	}
-	maxMemMegaBytes := uint64(500)
+	maxMemMB := uint64(500)
```

Comment:
It has always been active, but only for runtime memory. This adds a second check for total memory usage.

We were using over 1GB on many systems (around 80GB total on all systems) due to a WMI leak, which could have caused other systems to fail if we hadn't noticed.